### PR TITLE
Add Binding property to Saml2AuthenticationClass so it can be set 

### PR DESCRIPTION
**Pull request by ASupinski**
As described in https://github.com/KentorIT/authservices/issues/466

### CODE REVIEW

**Comment by ASupinski** [comment]
So... could I get a little guidance?  The checks are failing on my pull request saying I reduced the code coverage on Kentor.AuthServices/XmlHelpers.cs but if you look at my request I didn't alter that file at all and all of my test changes were to add new tests so I didn't accidentally remove coverage from an existing test... any suggestions?  Thanks!

**Comment by explunit** [comment]
It looks like a test is failing on appveyor (https://ci.appveyor.com/project/KentorIT/authservices/build/77) and this is why the coverage dropped. But I don't know why it's failing -- with current master all tests pass locally for me.

**Comment by explunit** [comment]
This appears to be related to .Net 4.6.2 -- probably AppVeyor has this installed now.

Created #545 as one option for resolving this.

**Comment by ASupinski** [comment]
Thanks for looking into it explunit!

**Comment by AndersAbel** [comment]
Closing and reopening again to cause a rebuild now that #545 is merged.

File: Kentor.AuthServices.Tests/Saml2P/Saml2AuthenticationRequestTests.cs [hunk 1]
```diff
@@ -416,5 +428,16 @@ private void Saml2AuthenticationRequest_ToXElement_AddsRequestedAuthnContextUtil
...
+
+        private void Saml2AuthenticationRequest_ToXElement_AddsProtocolBinding(AuthServices.WebSso.Saml2BindingType protocolBinding, string expectedProtocolBinding)
+        {
+            var subject = new Saml2AuthenticationRequest()
+            {
+                AssertionConsumerServiceUrl = new Uri("http://destination.example.com"),
+                Binding = protocolBinding
+            }.ToXElement();
+
+            subject.Attribute("ProtocolBinding").Value.Should().Equals(expectedProtocolBinding);
```
Comment (AndersAbel) [inline Kentor.AuthServices.Tests/Saml2P/Saml2AuthenticationRequestTests.cs]: In fluent assertions you should use `.Be` and not `.Equals`. The former throws an exception if they are not equal, the latter returns a value that is ignored. I'll fix when merging.

**Pull request merged**